src/internal/timer.rs

```diff
@@ -1,51 +1,45 @@
-use chrono::{DateTime, Duration, Utc};
-use std::time::Duration as StdDuration;
-use tokio::time::delay_for;
+use tokio::time::{delay_until, Duration, Instant};
 
 #[derive(Debug)]
 pub struct Timer {
-    due: DateTime<Utc>,
+    due: Instant,
     duration: Duration,
 }
 
 impl Timer {
+    /// construct timer, initially set and due `duration_in_ms` in the future
```

Comment:
Doc-strings should be written as proper sentences. They should start with an uppercase letter, and end with a full stop.

```suggestion
    /// Construct a `Timer`, which is initially set to expire in due `duration_in_ms` time from the current instant.
```